worlds/terraria/docs/setup_en.md

```diff
@@ -7,16 +7,28 @@ and [TModLoader](https://store.steampowered.com/app/1281930/tModLoader/) on Stea
 
 ## Installing the Archipelago Mod
 
-Subscribe to [the mod](https://steamcommunity.com/sharedfiles/filedetails/?id=2922217554) on Steam.
+1. Subscribe to [the mod](https://steamcommunity.com/sharedfiles/filedetails/?id=2922217554) on Steam
+2. Open TModLoader
```

Comment:
tModLoader is spelled with a lowercase t

Reply:
e9183cb7cbe6d0d2a64eae3cd82785d0543d90bf fixes this typo  throughout the guide.
tModLoader was capitalized in the guide when I found it, so I followed that. All previous mentions of "TModLoader" have been eliminated.